lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -692,6 +693,37 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return out.join( ',' );
 	});
 
+	/**
```

Comment:
@aayush0325 Re-adding the comment: namely, this should be moved to L461 so that methods are defined in alphabetical order.